.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Fixes _URL of issue_.
+
+_The first line of your PR should begin with the magic word "Fixes", followed by the full URL of the issue it is addressing. Every pull request must be linked to an issue on github.com/backdrop/backdrop-issues, and most of the rationale, discussion, and explanation should take place in the issue, rather than in the PR._
```

Comment:
```suggestion
<!-- The first line of your PR should begin with the magic word "Fixes", followed by the full URL of the issue it is addressing. Every pull request must be linked to an issue on github.com/backdrop/backdrop-issues, and most of the rationale, discussion, and explanation should take place in the issue, rather than in the PR. -->
```

Putting text in a hidden comment means people don't have to worry about removing it themselves, and it won't be shown in the final PR.